.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ orbs:
 
 jobs:
   test:
+    resource_class: large
```

Comment:
the large resource is included in our CircleCI plan, and should be faster than the default medium.